tests/ecs.rs

```diff
@@ -0,0 +1,27 @@
+//#![cfg(feature = "ecs")]
+
+extern crate rusoto;
+
+use rusoto::ecs::ECSClient;
+use rusoto::credentials::DefaultAWSCredentialsProviderChain;
+use rusoto::regions::Region;
+
+#[test]
+fn main() {
+    let credentials = DefaultAWSCredentialsProviderChain::new();
+    let region = Region::UsEast1;
+    let mut ecs = ECSClient::new(
+        credentials,
+        &region
+    );
+    match ecs.list_clusters(&Default::default()) {
```

Comment:
I think the list_clusters command needs to be passed in a ListClustersRequest object.  

`match ecs.list_clusters(&ListClustersRequest::default()) {`